src/core/Log.cpp

```diff
@@ -15,6 +15,11 @@
 #include <fmt/format.h>
 #include <fmt/printf.h>
 
+#ifdef _MSC_VER
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#endif
+
```

Comment:
I know this is WIP code not meant to wind up in the final version, but I'd strongly recommend against making this part of the Log file. Instead, both this and the stdout/stderr selection code should be wrapped in platform specific `OS::PrintDebugMessage()` implementations (one in `src/win32/OSWin32.cpp` and one in `src/posix/OSPosix.cpp`) which Log.cpp calls - I can write the POSIX implementation if you're interested in splitting this into another PR.

Reply:
Ahh, yes, sorry this is just a patch I add to local working versions to get logging output in MSCV easily.  I looked and I thought there might already be a place to add the platform specific logging but hadn't got around to it.

I'm happy to push a separate PR and tidy this up as I expect others may find it useful.

Reply:
That would be much appreciated!